core/voice/TalkManager.py

```diff
@@ -65,21 +89,28 @@ def getTexts(self, module, talk, strType='default') -> list:
 		return arr
 
 
-	def chooseTalk(self, talk: str, module: str, activeLanguage: str, defaultLanguage: str, shortReplyMode: bool) -> str:
-		try:
-			talkData = self._langData[module][activeLanguage][talk]
+	def _selectSentence(self, talkData: dict, shortReplyMode: bool) -> Union[str, Callable]:
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/ProjectAlice/ProjectAlice/pullRequest?prid=4626272)